editor/components/CrashRecommendationCard.tsx

```diff
@@ -130,7 +123,25 @@ export default function CrashRecommendationCard({
           crash_pk,
           created_by: user?.email,
           recommendations_partners: [],
-        },
+        };
+  }, [recommendation]);
+
+  /**
+   * Reset the form values after the recommendation is saved
+   */
+  useEffect(() => {
+    reset(defaultValues);
+  }, [defaultValues]);
```

Comment:
this was the missing piece: we need to `reset()` the form with fresh defaults after the mutation resolves

Reply:
cool! this was a nice refresher for me on resetting in RHF 🙏